core/src/main/scala/kafka/utils/LogDirUtils.scala

```diff
@@ -0,0 +1,63 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kafka.utils
+
+import kafka.controller.{LogDirEventNotificationListener}
+import scala.collection.{Map, Seq, Set}
+
+object LogDirUtils extends Logging {
+
+  private val LogDirEventNotificationPrefix = "log_dir_event_"
+  val LogDirFailureEvent = 1
```

Comment:
Initially I think it is OK not to check it since there is only one type. It is similar to how we currently have a version field in the reassignment json file provided to `kafka-reassign-partitions.sh` but we currently don't check that version either.

But it is also reasonable to check it. I have updated the code to throw IllegalArgumentException is the event type value is not 1.